src/containers/Airdrop/AirdropGiftTracker/sections/LoggedIn/segments/Invite/Invite.tsx

```diff
@@ -92,7 +92,7 @@ export default function Invite() {
                             i18nKey="nextTierBonus"
                             components={{ strong: <strong /> }}
                             values={{
-                                count: ` ${friendsRemaining} `,
+                                count: ` ${friendsRemaining} friends `,
```

Comment:
maybe the `friends` should go into the translation key directly though?

Reply:
good call, done